### PR TITLE
Use event-store http middleware

### LINKED ISSUE #34: [RFC] Split into PSR-15 middleware and stand-alone app
I'd like to split the package. The API middlewares should be moved to a library package so that you can composer install them in your own PSR-7/PSR-15 driven application and mix them with auth middleware and so on.

This package can be turned into a skeleton that uses the lib and provides a ready-to-use expressive app as it is now.

The lib can be called `event-store-http-middleware` or we integrate it in the [http-middleware](https://github.com/prooph/http-middleware) package.

Thoughts @prolic @sandrokeil @oqq @basz

### PR DESCRIPTION
**Pull request by codeliner**
Closes #34

### CODE REVIEW

File: composer.json.dist [hunk 1]
```diff
@@ -25,29 +25,29 @@
     ],
     "require": {
         "php": "^7.1",
-        "guzzlehttp/guzzle": "^6.3",
-        "php-http/guzzle6-adapter": "^1.1.1",
         "psr/container": "^1.0",
-        "prooph/event-store" : "^7.3.1",
-        "prooph/pdo-event-store": "^1.6",
+        "prooph/event-store" : "^7.2",
```
Comment (prolic) [inline composer.json.dist]: Why so many downgrades here?

File: config/config.php [hunk 2]
```diff
@@ -10,18 +10,23 @@
 
 declare(strict_types=1);
 
-namespace Prooph\EventStore\Http\Api;
+namespace Prooph\ProophessorDo;
```
Comment (prolic) [inline config/config.php]: ooops

File: config/development.config.php.dist [hunk 3]
```diff
@@ -1,4 +1,6 @@
 <?php
+
+declare(strict_types=1);
```
Comment (prolic) [inline config/development.config.php.dist]: we always did comments first, declare second

**Review by prolic — CHANGES_REQUESTED** [review]

File: composer.json.dist (hunk 1 above)
Comment (codeliner) [reply composer.json.dist]: uups, c & p from prooph-do ...

File: config/config.php (hunk 2 above)
Comment (codeliner) [reply config/config.php]: and another c&p issue ;)

File: config/development.config.php.dist (hunk 3 above)
Comment (codeliner) [reply config/development.config.php.dist]: and again, c&p

all fixed with latest push

**Review by prolic — APPROVED** [review]

**Pull request merged**